presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java

```diff
@@ -194,6 +215,9 @@ public TaskExecutor(
                 guaranteedNumberOfDriversPerTask,
                 maximumNumberOfDriversPerTask,
                 taskPriorityTracking,
+                DEFAULT_INTERRUPT_RUNAWAY_SPLITS_TIMEOUT,
+                DEFAULT_INTERRUPTIBLE_SPLIT_PREDICATE,
+                DEFAULT_INTERRUPT_SPLIT_INTERVAL,
```

Comment:
If these defaults are not going to be changed, I guess we don't need to pass them in through parameters? Just directly use them in the body of the class?

Reply:
Different ones are passed in for unit tests so the tests can complete quickly.